src/js/actions/layers.js

```diff
@@ -768,55 +769,87 @@ define(function (require, exports) {
             return Promise.resolve();
         }
 
+        if (modifier !== undefined && layerSpec.size > 1) {
```

Comment:
Yeah, but if you pass "select" with a big layerSpec here, it's going to complain at us. And that should be allowed. You can later on do the same check to set the pivot only if it was actually passed. Or we need to document somewhere that if you want this action to select multiple layers, don't provide any modifiers (even though select is a perfectly valid one)